inst/extdata/re-theme.json

```diff
@@ -47,14 +50,16 @@
       "size": 11,
       "fontFace": "plain",
       "fontFamily": "",
-      "angle": 0
+      "angle": 0,
+      "align": "center"
     },
     "yAxis": {
       "color": "black",
       "size": 11,
       "fontFace": "plain",
       "fontFamily": "",
-      "angle": 0
+      "angle": 0,
+      "align": "right"

```

Comment:
xAxis and yAxis correspond to x and y tick labels
They are named that way in the theme because it allows to directly map the `font` field of `xAxis` and `yAxis` objects of the plot configuration.

The difference in alignment of the y ticklabels is mainly seen in plots such as the sensitivity plots where the ticklabels are quite long. In that case, right alignment looks prettier in my opinion.

**Right aligned**

![image](https://user-images.githubusercontent.com/45343665/165920383-06fc87fc-93b7-4239-83bb-f30b92abe5bb.png)

**Center aligned**

![image](https://user-images.githubusercontent.com/45343665/165920488-d4c23458-eae9-49c5-a141-cb57dbcfa611.png)


**Left aligned**

![image](https://user-images.githubusercontent.com/45343665/165920545-210a9bf2-6ba7-4861-8b2f-4a35f63fa0e1.png)

Reply:
ok, got it :)
Agree: right alignment looks better in that case